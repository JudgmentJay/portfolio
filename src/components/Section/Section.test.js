import React from 'react'
import renderer from 'react-test-renderer'

import { Section } from '../'

const mockData = {
	id: 'mockId',
	header: 'Mock Header'
}

it('Renders correctly', () => {
	const tree = renderer
		.create(
			<Section {...mockData}>
				<div>This is a test section</div>
			</Section>
		)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
