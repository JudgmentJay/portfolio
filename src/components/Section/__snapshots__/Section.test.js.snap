// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<section
  className="section"
  id="mockId"
>
  <div
    className="container"
  >
    <h3
      className="sectionHeader"
    >
      Mock Header
    </h3>
    <div>
      This is a test section
    </div>
  </div>
</section>
`;
